fhd_core/source_modeling/fhd_diffuse_model.pro

```diff
@@ -52,6 +52,17 @@ IF size(diffuse_spectral_index,/type) EQ 10 THEN BEGIN ;check if pointer type
     print,"A spectral index is defined in the saved diffuse model, but this is not yet supported!"
 ENDIF ;case of specifying a single scalar to be applied to the entire diffuse model is treated AFTER building the model in instrumental polarization
 
+;if keyword_set(select_radius) THEN BEGIN
+;    ang2vec,obs.obsdec,obs.obsra,cen_coords,/astro
+;    Query_disc,nside,cen_coords,select_radius,inds_select,npix_sel,/deg
+;    pix2vec_ring, nside, inds_select, pix_coords
+;    vec2ang,pix_coords,pix_dec,pix_ra,/astro
+;    IF coord_use EQ 'galactic'   THEN glactc,pix_ra,pix_dec,2000.,pix_ra,pix_dec,2, /degree
+;    IF coord_use EQ 'equatorial' THEN Hor2Eq,pix_dec,pix_ra,obs.JD0,pix_ra,pix_dec,lat=obs.lat,lon=obs.lon,alt=obs.alt,precess=1,/nutate
+;    ang2vec, pix_dec, pix_ra, vecs, /astro
+;    vec2pix_ring, nside, vecs, hpx_inds
+;ENDIF
```

Comment:
Don't add commented-out code to `master`. If this is not added, then also don't add the keyword `select_radius` above.